sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubBufferedProducerClient.cs

```diff
@@ -100,13 +119,42 @@ internal class EventHubBufferedProducerClient : IAsyncDisposable
         public string Identifier => _producer.Identifier;
 
         /// <summary>
-        ///   <c>true</c> if the producer has been closed <c>false</c> otherwise.
+        ///   Indicates whether or not this <see cref="EventHubBufferedProducerClient" /> is currently
+        ///   active and publishing queued events.
         /// </summary>
         ///
+        /// <value>
+        ///   <c>true</c> if the client is publishing; otherwise, <c>false</c>.
+        /// </value>
+        ///
+        /// <remarks>
+        ///   The producer will begin publishing when an event is enqueued and should remain active until
+        ///   either <see cref="CloseAsync" /> or <see cref="DisposeAsync" /> is called.
+        ///
+        ///   If any events were enqueued, <see cref="IsClosed" /> is <c>false</c>, and <see cref="IsPublishing" />
+        ///   is <c>false</c>, this likely indicates an unrecoverable state for the client.  It is recommended to
+        ///   close the <see cref="EventHubBufferedProducerClient" /> and create a new instance.
+        ///
+        ///   In this state, exceptions will be reported by the Event Hubs client library logs, which can be captured
+        ///   using the <see cref="Azure.Core.Diagnostics.AzureEventSourceListener" />, as demonstrated in
+        ///   <seealso href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample10_AzureEventSourceListener.md">this</seealso>
```

Comment:
Good thought.   The sample is fairly thorough, with multiple snippets.   Since they're not directly related to the producer and use of the decorated property, they all felt out of place here and more distracting than helpful.   I refactored a bit and bumped the `<seealso>` into a stand alone block.